comptests/motr/test_motr_di.py

```diff
@@ -266,3 +273,59 @@ def test_data_block_corruption_one_by_one(self):
             # Read the data using m0cp utility
             self.m0cat_md5sum_m0unlink(bsize_list, count_list, layout_ids, object_list,
                                        client_num=client_num)
+
+    @pytest.mark.skip(reason="Test incomplete without teardown")
+    @pytest.mark.tags("TEST-42910")
+    @pytest.mark.motr_di
+    def test_m0cp_block_corruption_m0cat_degraded_mode(self):
```

Comment:
As discussed, Edit the deployment file using below method in the setup method. 
edit_deployments_for_delay

Reply:
need to change as per v0.09.0

Reply:
will pick in RBF task